scout/server/blueprints/clinvar/controllers.py

```diff
@@ -26,18 +27,15 @@
 LOG = logging.getLogger(__name__)
 
 
-def _get_var_tx_hgvs(case_obj, variant_obj):
-    """Retrieve all transcripts / hgvs for a given variant
-    Args:
-        case_obj(scout.models.Case)
-        variant_obj(scout.models.Variant)
-    Returns:
-        list of tuples. example: [("NM_002340.6:c.1840C>T", "NM_002340.6:c.1840C>T (validated)" ), ("NM_001145436.2:c.1840C>T", "NM_001145436.2:c.1840C>T"), .. ]
-    """
+def _get_var_tx_hgvs(case_obj: dict, variant_obj: dict) -> List[Tuple[str, str]]:
+    """Retrieve all transcripts / hgvs for a given variant."""
+
     build = str(case_obj.get("genome_build", "37"))
     tx_hgvs_list = [("", "Do not specify")]
+    add_gene_info(store, variant_obj, genome_build=build)
```

Comment:
This adds more info to the transcripts available in the variants. @Jakob37 hope this works! 🤞🏻

Reply:
I'll give it a try!